packages/manager/modules/pci/src/projects/project/ai/apps/app.constants.js

```diff
@@ -49,6 +49,27 @@ export const APP_STORAGE_INFO = {
   CA: 'https://docs.ovh.com/ca/en/publiccloud/ai/data/',
 };
 
+export const SPECIAL_CONDITIONS = {
```

Comment:
Are these specific to AI ?

Reply:
Seems to concern whole pci project, I moved it to the pci constants file : https://github.com/ovh/manager/pull/7228/commits/3a107be66d2de7407c8b0a26bbe39559831a859a